admin/widgets.rb

```diff
@@ -0,0 +1,171 @@
+
+  filter :title, label: I18n.t('active_admin.widget.title')
+  filter :css_name, label: I18n.t('active_admin.widget.css_class')
+  filter :id_name, label: I18n.t('active_admin.widget.id')
+  filter :sorter, label: I18n.t('active_admin.widget.sorter')
+
+  scope I18n.t('active_admin.widget.scope1'), :all, default: true
+  scope I18n.t('active_admin.widget.scope2'), :active
+  scope I18n.t('active_admin.widget.scope3'), :inactive
+  scope I18n.t('active_admin.widget.scope4'), :default
+
+  if ActiveRecord::Base.connection.table_exists?("tags")
+    Goldencobra::Widget.tag_counts_on(:tags).map(&:name).each do |wtag|
+      scope(I18n.t(wtag, scope: [:goldencobra, :widget_types], default: wtag).capitalize){ |t| t.tagged_with(wtag) }
+    end
+  end
+
+  form html: { enctype: "multipart/form-data" } do |f|
+    f.actions
+    f.inputs I18n.t('active_admin.widget.general'), class: "foldable inputs" do
+      f.input :title, label: I18n.t('active_admin.widget.label1'), hint: I18n.t('active_admin.widget.hint1')
+      f.input :tag_list, label: I18n.t('active_admin.widget.label2'), hint: I18n.t('active_admin.widget.hint2')
+      f.input :active, label: I18n.t('active_admin.widget.label3'), hint: I18n.t('active_admin.widget.hint3')
+      f.input :default, label: I18n.t('active_admin.widget.label4'), hint: I18n.t('active_admin.widget.hint4')
+    end
+    f.inputs I18n.t('active_admin.widget.layout_web'), class: "foldable inputs" do
+      f.input :content, label: I18n.t('active_admin.widget.label_web'), hint: I18n.t('active_admin.widget.hint_web')
+    end
+    # Wird so gut wie nie benutzt => erstmal ausgeblendet:
+    # f.inputs I18n.t('active_admin.widget.layout_mobile'), class: "foldable inputs closed" do
+    #   f.input :mobile_content, label: I18n.t('active_admin.widget.label_mobile'), hint: I18n.t('active_admin.widget.hint_mobile')
+    # end
+    f.inputs I18n.t('active_admin.widget.info'), class: "foldable inputs closed"  do
+      f.input :sorter, label: I18n.t('active_admin.widget.label_info1'), hint: I18n.t('active_admin.widget.label_hint1')
+      f.input :css_name, label: I18n.t('active_admin.widget.label_info2'), hint: I18n.t('active_admin.widget.label_hint2')
+      f.input :id_name, label: I18n.t('active_admin.widget.label_info3'), hint: I18n.t('active_admin.widget.label_hint3')
+      # Wird so gut wie nie benutzt => erstmal ausgeblendet:
+      #f.input :teaser
+      #f.input :description, label: I18n.t('active_admin.widget.label_info4'), hint: I18n.t('active_admin.widget.label_hint4')
+    end
+    f.inputs I18n.t('active_admin.widget.access_rights'), class: "foldable closed inputs" do
+      f.has_many :permissions do |p|
+        p.input :role, include_blank: "Alle"
+        p.input :action, as: :select, collection: Goldencobra::Permission::PossibleActions, include_blank: false
+        p.input :_destroy, as: :boolean
+      end
+    end
+    f.inputs I18n.t('active_admin.widget.article') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.